### DISCUSSION TITLE
User Dictionary Import feature missing in mozc for Android

### DISCUSSION THREAD

**Discussion by helpthetibetians** [google/mozc#884] [Q&A]
From what I know, mozc for Android has a feature for exporting user dictionary but it does not have a feature for "importing" a text file into a user dictionary.

Has anyone already implemented the import feature?

**Comment by yukawa** [comment]
As explained in #879, this repository no longer contains Android client Java code. If you are interested in contributing to forked versions, you can reach out to the owners of those forked versions.
